### DISCUSSION TITLE
How to allow windows to overlap the bar

### DISCUSSION THREAD

**Discussion by mcnuggetsx20** [qtile/qtile#3231] [Q&A]
I want to make something like a desktop panel and since there is no dedicated way to do it in qtile, I thought I could put another bar on the screen, which would be on the desktop, behind the opened windows. But the problem is, windows are trying not to overlap it which clearly breaks the idea. Is there any way to allow windows to overlap a certain bar?

**Comment by m-col** [comment]
This is due to a `_NET_WM_STRUT_PARTIAL` hint on the bar window. If this is not set, then the window won't be given some reserved space. Perhaps you can see disable that hint.

**Reply by mcnuggetsx20** [reply]
Sounds great but how do I do that?

**Reply by m-col** [reply]
This would be up to the panel that you use. Maybe they have a setting to disable the hint, e.g. for `polybar` this seems to be the `margin-X` settings.
